SU-HSDO/suhumsci
# feat(STN-136):  polish horizontal card

# READY FOR REVIEW 🛑 

## Summary
**Card**: [STN-136](https://sparkbox.atlassian.net/browse/STN-136)
**Author**: @meranicosme 

Polishes and adds refinements for horizontal card, external link, and image zoom hover effect.


## Steps to Test
1. Check out the horizontal card locally. Make sure behavior and style is as expected.
2. In the `humsci_basic` directory, run `npm test` and make sure everything passes.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
